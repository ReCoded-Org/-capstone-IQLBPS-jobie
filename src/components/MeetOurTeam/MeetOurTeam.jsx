import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faLinkedin } from "@fortawesome/free-brands-svg-icons";

function MeetOurTeam({ teamMembers }) {
  const memberCard = teamMembers.map((member) => {
    return (
      <div className="m-4" key={member.name}>
        <img
          className="rounded-full"
          src={member.src}
          alt={`${member.name} png`}
        />
        <h3 className="text-3xl font-semibold text-accent m-2 text-center">
          {member.name}
        </h3>
        <h2 className="text-lg text-gray text-center">
          {member.position}
          <span className="m-2">
            <a href={member.linkedin} rel="noreferrer" target="_blank">
              <FontAwesomeIcon
                icon={faLinkedin}
                className="hover:text-blue-500 "
              />
            </a>
          </span>
        </h2>
      </div>
    );
  });
  return (
    <section className="bg-gray-500/5">
      <h1 className="text-5xl text-secondary text-center py-8 font-semibold">
        Meet Our <span className="text-accent">Team</span>
      </h1>
      <div className="grid md:grid-cols-3 place-content-center gap-x-20 gap-y-8 px-10">
        {memberCard}
      </div>
    </section>
  );
}

export default MeetOurTeam;
