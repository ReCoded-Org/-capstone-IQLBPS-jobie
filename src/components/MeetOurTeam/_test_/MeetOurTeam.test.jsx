import renderer from 'react-test-renderer';
import MeetOurTeam from '../MeetOurTeam.jsx'

it('renders Meet Our Team component correctly', () => {
    const tree = renderer
      .create(<MeetOurTeam/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });