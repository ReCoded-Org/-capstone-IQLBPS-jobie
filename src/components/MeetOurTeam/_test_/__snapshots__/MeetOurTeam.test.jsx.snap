// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Meet Our Team component correctly 1`] = `
<section
  className="bg-gray-500/5"
>
  <h1
    className="text-5xl text-secondary py-8 font-semibold"
  >
    Meet Our 
    <span
      className="text-accent"
    >
      Team
    </span>
  </h1>
  <div
    className="grid md:grid-cols-3 gap-x-20 gap-y-8 px-10"
  >
    <div
      className="m-4"
    >
      <img
        alt="Allan Ahmed png"
        className="rounded-full"
        src="/assets/meetOurTeam/allan.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Allan Ahmed
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/allan-ahmed/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Iman Zuher png"
        className="rounded-full"
        src="/assets/meetOurTeam/iman.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Iman Zuher
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/iman-zuher/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Ahmed Rashid png"
        className="rounded-full"
        src="/assets/meetOurTeam/ahmed.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Ahmed Rashid
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/ahmed-rashid-8a9b9116b/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Noora Hakim png"
        className="rounded-full"
        src="/assets/meetOurTeam/noora.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Noora Hakim
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/noora-hakim-156144244/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Hakam Shakir png"
        className="rounded-full"
        src="/assets/meetOurTeam/hakam.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Hakam Shakir
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/hakam-shakir-31b358204/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Marwa Salameh png"
        className="rounded-full"
        src="/assets/meetOurTeam/marwa.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Marwa Salameh
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/marwa-salameh/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Zakaria Ahmad png"
        className="rounded-full"
        src="/assets/meetOurTeam/zakaria.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Zakaria Ahmad
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/zakaria-ahmad-76754a239/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Parykhan Jameel png"
        className="rounded-full"
        src="/assets/meetOurTeam/parykhan.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Parykhan Jameel
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/parykhan-jameel-5b5078241/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
    <div
      className="m-4"
    >
      <img
        alt="Lawen Sardar png"
        className="rounded-full"
        src="/assets/meetOurTeam/lawen.jpg"
      />
      <h3
        className="text-3xl font-semibold text-accent m-2"
      >
        Lawen Sardar
      </h3>
      <h2
        className="text-lg text-gray"
      >
        Frontend Developer
        <span
          className="m-2"
        >
          <a
            href="https://www.linkedin.com/in/lawen-sardar/"
            rel="noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-linkedin hover:text-blue-500 "
              data-icon="linkedin"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </span>
      </h2>
    </div>
  </div>
</section>
`;
